docs/release.md

```diff
@@ -15,11 +15,6 @@ changelog is updated by developers via the normal pull request process.
 
 Releases can be found on the github [release page](https://github.com/CSCfi/rems/releases).
 
-## Environments
-
-The integration test environment is updated by the developers.
-CSC updates all supported environments such as https://rems2demo.csc.fi which is for public testing and customer specific instances.
```

Comment:
This phrase is a little difficult to parse.

Reply:
I deleted the Environments stuff from here because the demo environments etc. are mentioned in the README